src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -452,6 +452,66 @@
     </widget>
    </item>
 
+   <item>
+    <widget class="QGroupBox" name="groupBox_cover_art_fetcher">
+     <property name="title">
+      <string>Cover Art Fetcher</string>
```

Comment:
"Tag Fetcher: Preferred Cover Art Resolution"?
or
"Tag Fetcher: Preferred Cover Art Size"?
the the radio buttons could be labeled `XXL` / `XL` / `M` / `S` (is this more clear in terms of internationaization?) and would fit into one row.
Just an idea..

Reply:
dismissed, see my comment https://github.com/mixxxdj/mixxx/pull/4851#issuecomment-1321244868